WalletWasabi/WabiSabi/Client/AliceClient.cs

```diff
@@ -114,6 +114,14 @@ private static async Task<AliceClient> RegisterInputAsync(RoundState roundState,
 						Logger.LogInfo($"{coin.Coin.Outpoint} is banned.");
 						break;
 
+					case WabiSabiProtocolErrorCode.InputLongBanned:
+						var split = wpe.Message.Split(" ");
+						var bannedUntilUnixSeconds = long.Parse(split[^1]);
```

Comment:
I don't like this. I will try to find a better solution, so we don't have to split a string and etc.

Reply:
We need that remaining time in both ban cases. I would start by changing the response first and

Reply:
I changed it, but still I'm not satisfied with the solution.
After [a47db90](https://github.com/zkSNACKs/WalletWasabi/pull/7964/commits/a47db90b2ceee1f40437bab4d57aa0cdaa440778) I simply put the remaining time into the Exception Message prop. 
```
throw new WabiSabiProtocolException(WabiSabiProtocolErrorCode.InputLongBanned, $"{Config.ReleaseUtxoFromPrisonAfterLongBan - inmate.TimeSpent}");
```

But in the future if we want to send more info, this solution won't hold up.